packages/common/test/content/content.test.ts

```diff
@@ -1410,4 +1414,147 @@ describe("content: ", () => {
       });
     });
   });
+
+  // Can be found in packages/common/src/content/contentUtils.ts
+  describe("getContentStatus", () => {
+    it("entity passed in does not have a url and is unavailable", async () => {
+      const entity: IHubEditableContent = {
+        id: "abc",
+        url: undefined,
+
+        // not important to this test
+        licenseInfo: "",
+        itemControl: "",
+        owner: "",
+        schemaVersion: 0,
+        tags: [],
+        canEdit: false,
+        canDelete: false,
+        name: "",
+        createdDate: new Date(),
+        createdDateSource: "",
+        updatedDate: new Date(),
+        updatedDateSource: "",
+        type: "",
+        orgUrlKey: "",
+      };
+
+      const result = await getContentStatus(entity, MOCK_REQUEST_OPTIONS);
+      expect(result.service.availability).toBe("unavailable");
+    });
+
+    it("service wins race and is available", async () => {
+      const entity: IHubEditableContent = {
+        id: "abc",
+        url: "https://hubqa.arcgis.com/api/v3/connectors/test/file-geojson/rest/services/slowpoints/FeatureServer/0?stop=false",
+
+        // not important to this test
+        licenseInfo: "",
+        itemControl: "",
+        owner: "",
+        schemaVersion: 0,
+        tags: [],
+        canEdit: false,
+        canDelete: false,
+        name: "",
+        createdDate: new Date(),
+        createdDateSource: "",
+        updatedDate: new Date(),
+        updatedDateSource: "",
+        type: "",
+        orgUrlKey: "",
+      };
+
+      fetchMock.once(
+        "https://hubqa.arcgis.com/api/v3/connectors/test/file-geojson/rest/services/slowpoints/FeatureServer/0?stop=false",
+        new Promise((resolve) => {
```

Comment:
As we discussed, it would be best to use the built-in [delay](https://www.wheresrhys.co.uk/fetch-mock/api/mocking/parameters/options/#delay) option, though I know it's a hassle b/c of the outdated types.